src/widgets/VaccineChart.js

```diff
@@ -131,15 +200,21 @@ export class VaccineChart extends React.Component {
           maxDomain={{ y: maxDomain }}
           minDomain={{ y: minDomain }}
         >
-          <VictoryAxis offsetY={50} />
-          <VictoryAxis dependentAxis offsetX={50} crossAxis={false} />
-          {minTempPlotLine}
-          {maxTempPlotLine}
-          {minLinePlotLine}
-          {minLinePlotScatter}
-          {maxLinePlotLine}
-          {maxLinePlotScatter}
-          {hazardPlotScatter}
+          <VictoryAxis offsetY={50} style={{ tickLabels: axisStyles }} />
+          <VictoryAxis
+            dependentAxis
+            offsetX={50}
+            crossAxis={false}
+            tickFormat={t => `${t}℃`}
+            style={{ tickLabels: axisStyles }}
+          />
+          {this.renderMinLogTemperatureLine()}
```

Comment:
opinion: `renderTempLines`, `renderScatters`, `renderBoundary`, they determine which to render and return depending on props and pass a value to `renderScatter()`, `renderLine`, `renderBoundary` -- don't do this though

Reply:
I like the idea of each component being handled clearly by its own render method. It makes the logic a bit "flatter". But this is probably tidier because its more modular 🤔 .